### DISCUSSION TITLE
Return hyperlinks in custom scripts output

### DISCUSSION THREAD

**Discussion by MajesticFalcon** [netbox-community/netbox#10336] [Q&A] [answered]
Looking through the code, it appears the script_result.htm(lx) page is rendered to the user once the script completes. Is it possible to return an HTML object instead of just plain text? The application would be that the custom script is to provide very specific searching capabilities and the output is a hyperlink to the correct object the user is looking for. Returning an "<a>" tag doesn't do what I hoped it would.

**Comment by kkthxbye-code** [comment, marked as answer]
The logs can contain markdown.

`self.log_success('[google](https://google.com)')`

![image](https://user-images.githubusercontent.com/400797/189686866-cc8cb467-0e99-4b22-bddc-528128e7542d.png)
